pageComponents/NewPostPage/HeroPositionSelect.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import Select, { Option } from "@@/components/Select";
+import { POSITION_CORRESPONDENCE_EACH_PLAYER_LENGTH } from "@@/utilities/getPositionByPlayerAndIndex";
+
+interface Props extends React.Attributes {
+  playerLength: number;
+  onChange?: (e: React.ChangeEvent<HTMLSelectElement>, value: string) => void;
```

Comment:
Does this return string value like "SB" or "MP2"? I think it would be better to return integer value as same as `heroIndex`.

I guess the reason why you chose string is <Select> only supports string values, but you should think how to deal with it, instead of propagating problem to wider area. Feel free to ask me if you need my help.